example/spec/testAPI/APISuccessSpec.js

```diff
@@ -174,7 +179,7 @@ describe('The Cumulus API', () => {
         workflow: 'PublishGranule'
       });
 
-      const granulePublished = await waitForExist(cmrLink, true, 6, 20000);
+      const granulePublished = await waitForExist(cmrLink, true, 20, 20000);
```

Comment:
will this address some of the issues we've been hitting in integration tests with granule publishing failing intermittently?

Reply:
I think so... it seems to happen more often on master? That's what I am trying to do here.